mix3d/pwa-kit
# Optimize visibility-off svg

# Description

Removes the slash, strangely created with an svg filter, in such a way that prevented the icon being used on anything other than a white background.

# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- Changes the SVG definition.
- old vs new:
![image](https://user-images.githubusercontent.com/1474137/161155707-c9bf8bd9-cd55-4aff-a3a5-8c7a097cca5a.png)



# How to Test-Drive This PR

- change the background color of the password field
- observe that the slash of the icon doesn't draw white.

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
